pkg/controller/spinnakervalidating/spinnakervalidating_controller.go

```diff
@@ -35,17 +38,20 @@ var log = logf.Log.WithName("spinvalidate")
 
 // Add adds the validating admission controller
 func Add(m manager.Manager) error {
-	ns, err := k8sutil.GetOperatorNamespace()
-	if err != nil {
-		return err
+	ns := os.Getenv("WATCH_NAMESPACE")
```

Comment:
Why `WATCH_NAMESPACE` here? When would the validation service be in a different namespace than the operator's?

Reply:
This was used to be able to run the admission webhook locally, taking the namespace from that variable instead of loading it from a file that exists in the pod. What could be a better way to get the namespace when running outside the pod?